.php_cs.dist

```diff
@@ -8,6 +8,8 @@ return PhpCsFixer\Config::create()
     ->setRules(array(
         '@Symfony' => true,
         '@Symfony:risky' => true,
+        '@PHPUnit48Migration:risky' => true,
+        'php_unit_no_expectation_annotation' => false, // part of `PHPUnitXYMigration:risky` ruleset, to be enabled when PHPUnit 4.x support will be dropped, as we don't want to rewrite exceptions handling twice
```

Comment:
is the comment accurate? shouldn't we remove it?
I mean: I'm not sure we're going to move to "annotations" first, since using them works fine and is quite common in the code base

Reply:
rule is not about move "to", it's about move "away" from annotations.
annotations were already marked as not best practice, as it's better to reference by code to given class and to mark what is expected to raise exception (so one can say "expecting exception from this line" instead "anywhere in whole test method")
so, it is recommended to use `->setExpected*` methods (which were removed on v6)
then, almost 2 years ago best practice was challenged again and changed to what we know from phpunit 5+ (`->expect*`) (not available on v4, note https://github.com/sebastianbergmann/phpunit/pull/2931)
please take a note: https://thephp.cc/news/2016/02/questioning-phpunit-best-practices

for that, it's part of migration ruleset, excluding rule out of the blue is always weird, it's better to describe why - I put a comment.
If you don't see it's usefull, feel free to drop it

Reply:
we're still maintaining branch 3.4 for a few years, and it runs on 5.5, so I'm not sure we can migrate the code base to `->setExpected*` at all, isn't it?
since we also don't want to generate numerous merge conflicts, we won't want to apply this to master/4.0
which means this comment is premature AFAIK, that's what I meant

Reply:
and this comment is exactly to say that thing - as we don't want to use `->setExpected*`, which is a temporary solution, we excluded this rule even if it's part of PHPUnit migration ruleset.

If you find it useless, simply drop it.